src/Screens/images/three-dot-menu.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="6" height="26" viewBox="0 0 6 26" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="3" cy="3" r="3" fill="green"/>
+<circle cx="3" cy="13" r="3" fill="blue"/>
+<circle cx="3" cy="23" r="3" fill="red"/>
+</svg>
```

Comment:
What is that green, blue red fill? IMO fill should be only black and then you can prop it to be somehting other?